packages/flutter/test/foundation/consolidate_response_test.dart

```diff
@@ -10,59 +10,81 @@ void main() {
     final List<int> chunkOne = <int>[0, 1, 2, 3, 4, 5];
     final List<int> chunkTwo = <int>[6, 7, 8, 9, 10];
     MockHttpClientResponse response;
-
+    
     setUp(() {
       response = new MockHttpClientResponse();
-       when(response.listen(
-         typed(any),
-         onDone: typed(any, named: 'onDone'),
-         onError: typed(any, named: 'onError'),
-         cancelOnError: typed(any, named: 'cancelOnError')
-      )).thenAnswer((Invocation invocation) {
-        final void Function(List<int>) onData = invocation.positionalArguments[0];
-        final void Function(Object) onError = invocation.namedArguments[#onError];
+      when(response.listen(typed(any),
+              onDone: typed(any, named: 'onDone'),
+              onError: typed(any, named: 'onError'),
+              cancelOnError: typed(any, named: 'cancelOnError')))
```

Comment:
please use flutter-style indentation here and elsewhere (if you're using dartfmt, it gets pretty close if you just make sure everywhere that can have a trailing comma has one)

Reply:
Hopefully that's better